### PR TITLE
Added a Reason enum and changed tests to accomadate for the enum

### PR DESCRIPTION
**Pull request by eekansh-M**
## Pull Request

[comment]: <> (Leave blank if a section does not apply.)

### Brief Summary
[comment]: <> I created the Reason enum and changed tests so that I can accommodate for the existence of this enum and made sure the methods that are available for comment status enums are working. 
- 
- 

### Questions / Considerations for the Future
[comment]: <> (Note any questions, or things to note that might involve this PR in the future.)
- Um, is there anything else I should do for this or was the ticket just creating the enum. 
- 

### API Changes
[comment]: <> (Note any new endpoints, deleted endpoints, or updated endpoints.)
- 
- 

### Database Changes
[comment]: <> (Note changes to any database schemas.)
- 
- 

### New Tests
[comment]: <> Created new tests to test for Reason enum.
- 
- 

[comment]: <> #250 
Closes #

### CODE REVIEW

**Review by ethanszeto — APPROVED** [review]
LGTM

**Review by suttonspindler — APPROVED** [review]
Looks good!

I think a name like DenialReason would be more intuitive, but I understand that's not as terse either.  The logic is good for now

**Comment by ethanszeto** [comment]
> Looks good!
> 
> I think a name like DenialReason would be more intuitive, but I understand that's not as terse either. At the very least the logic is good for now

Bro dismissed my review

**Comment by ethanszeto** [comment]
> Looks good!
> 
> I think a name like DenialReason would be more intuitive, but I understand that's not as terse either. At the very least the logic is good for now

It think the reasoning behind it was there might be an approval reason too? In any case these will be special comments with a very clear usage

**Review by ethanszeto — APPROVED** [review]

File: tests/__tests__/unit/enum/commentStatus.js
Comment (ethanszeto) [inline tests/__tests__/unit/enum/commentStatus.js:1]: next time write as a (nit), instead of making a change on branch, thanks

**Pull request merged**